documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
+In 360Giving data we ask publishers to use a list code taken from the [org-id list locator](http://org-id.guide/). This provides an open, maintained list of codes for many different lists around with world, giving a way to identify almost any organisation. 
 
-For example, a charity registered in England and Wales with the [Charity Commission](https://www.gov.uk/government/organisations/charity-commission) and a number of **1070468** will use an [IATI Organisation Registration Agency Code](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) of **GB-CHC**. The organisation identifier in your 360Giving data would be:
+```eval_rst
 
->GB-CHC-1070468
+.. admonition:: For example
 
-However, some organisations have more than one such identifier (e.g. where they are both a company and a charity), and others are not registered anywhere, and so you may only have names, or internal IDs, for them recorded on your data.
+  A charity registered in England and Wales with the Charity Commission of England and Wales, with the charity number '1070468' will use a prefix of ``GB-CHC``. 
 
-To give users of 360Giving data the best chance of joining up information about the same organisations across different data files, we follow a simple methodology to create Organisation Identifiers via the following process.
+  This gives an unique organisation identifier of ``GB-CHC-1070468``
 
-Work your way through the following steps. As soon as a step gives you an identifier, you can stop there and use the given identifier.
+```
 
-1. **Registered company number (UK)**
+### Choose the best identifier
 
-    If you have a registered company number for the organisation, use the Registration Agency Code 'GB-COH-' and the registered number (indicating that the number could be looked up at [Companies House](http://www.companieshouse.gov.uk))
+Some organisations have more than one identifier: they might be a charity **and** a company (charitable companies), or a charity **and** an educational establishment. 
 
-2. **Registered charity number (UK)**
+In these cases, it's important to know which identifier to pick so that users of data have the best possible chance of understanding that two grants have been made to the same organisation. 
 
-   If you have a **registered charity number** for the organisation, use the Registration Agency Code:
-    * 'GB-CHC-' for a [charity registered in England and Wales](http://www.charitycommission.gov.uk/), 
-    * 'GB-SC-' for a [charity registered in Scotland](http://www.oscr.org.uk/), or
-    * 'GB-NIC-' for a [charity registered in Northern Ireland](http://www.charitycommissionni.org.uk/) 
-    
-    along with the registered number.
+[org-id.guide](http://org-id.guide) ranks identifier lists by relevance and quality to help you pick the best identifier. 
```

Comment:
suggest updating to:
org-id.guide ranks identifier lists by relevance and quality to help you pick the best identifier, based on what you have available.

Reply:
Ok, that's sound - helps clarify that publishers aren't expected to use the top ranked if they don't have already.